File: C PROGRAMMING LANGUAGE/PROOF_OF_CONCEPTS/FUNCTIONS/TYPE-1/swap.c
void swap();
void main()
{
	swap();
}
void swap()
{
	int a,b;
	printf("enter two numbers:");
	scanf("%d%d",&a,&b);
	printf("\nbefore swappying numbers are %d and %d",a,b);
	int temp = a;
	a = b;
	b = temp;
	
	printf("\nafter swappying numbers are %d and %d",a,b);
}

File: JAVA/Swap.java
//Swap two numbers without using a third variable.

public class Swap
{
 public static void main(String[] args)
 {
  int a=10,b=20;
  System.out.println("before swappying : " +a+"  "+b);
  a = a+b;
  b = a-b;
  a = a-b;
  System.out.println("after swappying : " +a+"  "+b);
 }
}

/*
with using 3rd variable

int a = 10,b=20;
int temp = a;
a = b;
b = temp;

*/